src/mmw/js/src/modeling/models.js

```diff
@@ -15,9 +15,75 @@ var ResultCollection = Backbone.Collection.extend({
     model: ResultModel
 });
 
-var ProjectModel = Backbone.Model.extend({});
+var ProjectModel = Backbone.Model.extend({
+    urlRoot: '/api/modeling/projects/',
+
+    initialize: function() {
+        // TODO: For a new project, users will eventually
+        // be able to choose which modeling package
+        // they want to use in their project. For
+        // now, the only option is TR55, so it is
+        // hard-coded here. This will likely be a model
+        // itself in the future, with more modification
+        // and input control data.
+        this.set('model_package', 'tr-55');
+        this.set('active_scenario_slug', 'current-conditions');
+        this.set('taskModel', new Tr55TaskModel());
+    },
+
+    saveAll: function() {
+        var self = this;
+
+        // If we haven't saved the project before,
+        // hold on to the scenarios so that we can
+        // save each one once we have an ID for the project
+        if (this.isNew) {
+            var tempScenarios = this.get('scenarios');
+        }
+
+        this.save()
+            .done(function(resp) {
+                console.log('Saved project');
+
+                var scenarios = tempScenarios ? tempScenarios : this.get('scenarios');
+
+                scenarios.each(function(scenario) {
+                    if (!scenario.get('project')) {
+                        scenario.set('project', self.get('id'));
+                    }
+
+                    scenario
+                        .save()
+                        .done(function(resp) {
+                            console.log('Saved scenario');
+                        })
+                        .fail(function(resp) {
+                            console.log('Failed to save scenario');
+                        });
+                });
+            })
+            .fail(function(resp) {
+                console.log('Failed to save project');
+            });
+    },
+
+    parse: function(response, options) {
```

Comment:
I'd be more open to using the DRF add-on then doing that. Seems more fragile and that we would have to implement for every model that needs to be synced.